templates/github/.github/workflows/create-branch.yml.j2

```diff
@@ -27,7 +27,7 @@ jobs:
 
       - uses: actions/setup-python@v2
         with:
-          python-version: "3.8"
+          python-version: "{{ python_version }}"
```

Comment:
This workflow doesn't run Pulp at all so I don't see any benefit to changing this to `python_version` instead of 3.8.